platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -299,7 +299,7 @@ private boolean onVideoInit() {
 			for (GodotPlugin plugin : pluginRegistry.getAllPlugins()) {
 				plugin.onRegisterPluginWithGodotNative();
 			}
-			setKeepScreenOn("True".equals(GodotLib.getGlobal("display/window/energy_saving/keep_screen_on")));
+			setKeepScreenOn("true".equals(GodotLib.getGlobal("display/window/energy_saving/keep_screen_on")));
```

Comment:
Let's use [`Boolean.valueOf`](https://developer.android.com/reference/java/lang/Boolean#valueOf(java.lang.String)) instead. It performs a case-insensitive comparison which would avoid issues like that.